app/enterprise/1.3-x/deployment/installation/amazon-linux.md

```diff
@@ -1,321 +1,312 @@
   
-  if RBAC is enabled:
-  *http :8001/services Kong-Admin-Token:$KONG_PASSWORD
-  ```
-
-**curl**
-  ```
-  curl -i -X GET --url http://localhost:8001/services
-
-  if RBAC is enabled:
-  curl -i -X GET --url http://localhost:8001/services --header 'kong-admin-token:<KONG_PASSWORD>'
-  ```
-
-
-You should receive a `200 OK` response:
-
-```
-HTTP/1.1 200 OK
-Access-Control-Allow-Origin: *
-Connection: keep-alive
-Content-Length: 23
-Content-Type: application/json; charset=utf-8
-Date:
-Server: kong/kong-enterprise-edition
-X-Kong-Admin-Request-ID: 
-
-{
-    "data": [],
-    "next": null
-}
-```
-
-If you did not receive a `200` check out the troubleshooting section below or contact your Sales or Support Representative for assistance.
-
-
-## Troubleshooting
+    This setting needs to resolve to a network path that will reach the Amazon Linux host.
+  
+2. It is necessary to update the administration API setting to listen on the needed network interfaces on the CentOS host. A setting of `0.0.0.0:8001` will listen on port `8001` on all available network interfaces.
+  
+    ```
+    admin_listen = 0.0.0.0:8001, 0.0.0.0:8444 ssl
+    ```
+  
+3. You may also list network interfaces separately as in this example:
+  
+    ```
+    admin_listen = 0.0.0.0:8001, 0.0.0.0:8444 ssl, 127.0.0.1:8001, 127.0.0.1:8444 ssl
+    ```
+  
+4. Restart Kong for the setting to take effect:
 
-##### I receive an error when downloading the RPM Package
-  Ensure that you are using your Bintray Username and Bintray account **password **in the `wget` request - note that this is different from your Bintray API KEY.
+    ```bash
+    $ sudo /usr/local/bin/kong restart
+    ```
+  
+5. You may now access Kong Manager on port `8002`.
 
+### Enable the Developer Portal
 
-##### HTTP 401 - Unauthorized
-  If you have enabled RBAC all request must be sent with the `kong-admin-token` header 
+1. Kong Enterprise's **Developer Portal** can be connected by setting the `portal` property to `on` and setting the `portal_gui_host` property to the EC2 instance's `IPv4` address
 
-  **httpie**
     ```
-    Kong-Admin-Token:$KONG_PASSWORD
+    portal = on
+    portal_gui_host = <DNSorIP>:8003
+    ```
+
+2. Restart Kong for the setting to take effect:
+ 
+    ```bash
+    $ sudo /usr/local/bin/kong restart
     ```
 
-  **curl**
+3. The final step is to enable the Developer Portal. To do this, execute the following command, updating `DNSorIP` to reflect the IP or valid DNS for the Amazon Linux system.
+  
+    ```bash
+    $ curl -X PATCH http://<DNSorIP>:8001/workspaces/default   --data "config.portal=true"
+    ```
+  
+4. You can now access the Developer Portal on the default workspace with a URL like:
+  
     ```
-    --header 'kong-admin-token:<KONG_PASSWORD>'
+    http://<DNSorIP>:8003/default
     ```
 
+## Troubleshooting
+
+If you did not receive an `HTTP/1.1 200 OK` message, or need assistance completing
+setup reach out to your Kong Support contact or head over to the
```

Comment:
change this line to "your setup, reach out to your Kong Support representative or go to the"